domain/service/chatService.ts

```diff
@@ -1,3 +1,5 @@
 export default interface ChatService {
   sendMessageToChannel(message: string, channelId: string): void;
+  deleteMessageFromChannel(messageId: string, channelId: string): void;
+  sendDM(message: string, userId: string): void;
```

Comment:
Preferível ser mais explícito e evitar acrónimos.

```suggestion
  sendDirectMessageToUser(message: string, userId: string): void;
```

Reply:
feito